build/gulpfile.hygiene.js

```diff
@@ -247,6 +251,21 @@ function hygiene(some) {
 		this.emit('data', file);
 	});
 
+	// {{SQL CARBON EDIT}}
+	// Check for unnecessary 'use strict' lines. These are automatically added by the alwaysStrict compiler option so don't need to be added manually
+	const useStrict = es.through(function (file) {
+		const lines = file.__lines;
+		lines.forEach((line, i) =>  {
```

Comment:
You might want to only check the first few lines of the file. The typescript compiler will throw an error if you add 'use strict' after imports anyways, so the string will only appear directly after the copyright header. So maybe the first ~6 lines. Reduces the change of a false positive and speeds up the processing.